lib/membrane_h264_plugin/parser/nalu_parser.ex

```diff
@@ -3,42 +3,65 @@ defmodule Membrane.H264.Parser.NALuParser do
   A module providing functionality of parsing a stream of binaries, out of which each
   is a payload of a single NAL unit.
   """
+
+  alias Membrane.H264.Parser
   alias Membrane.H264.Parser.{NALu, NALuTypes}
-  alias Membrane.H264.Parser.NALuParser.SchemeParser
-  alias Membrane.H264.Parser.NALuParser.Schemes
+  alias Membrane.H264.Parser.NALuParser.{SchemeParser, Schemes}
+
+  @annexb_prefix_code <<0, 0, 0, 1>>
```

Comment:
we already have this attribute defined in `parser.ex` - it will be easier to be consistent if we define that prefix code in one place only

Reply:
I think since it won't change then it is cleaner to have it as a module attribute in both places than to try to share it with a function

Reply:
how about moving the `prefix_nalus` function to the `nalu_parser.ex`?